File: Local/etc/modulefiles/lua/core/matmodlab.lua
unload('anaconda/3.5')
load('macports')
load('gcc/6.2')
local HOME = os.getenv('HOME')
local prefix = pathJoin(HOME, '.conda/envs/matmodlab')
prepend_path('PATH', pathJoin(prefix, 'bin'))
setenv('CONDA_PREFIX', prefix)
setenv('CONDA_DEFAULT_ENV', 'matmodlab')

local DEVELOPER = os.getenv('DEVELOPER')
mml_userenv = pathJoin(DEVELOPER, 'Kayenta/hosts/matmodlab/mml_userenv.py')
if not isFile(mml_userenv) then
  LmodWarn('Kayenta user environment file not found')
else
  setenv('MML_USERENV', mml_userenv)
end

File: Local/etc/modulefiles/core/matmodlab.py
unload('anaconda/3.5')
load('macports')
load('gcc/6.2')
HOME = getenv('HOME')
prefix = os.path.join(HOME, '.conda/envs/matmodlab')
prepend_path('PATH', os.path.join(prefix, 'bin'))
setenv('CONDA_PREFIX', prefix)
setenv('CONDA_DEFAULT_ENV', 'matmodlab')

DEVELOPER = getenv('DEVELOPER')
mml_userenv = os.path.join(DEVELOPER, 'Kayenta/hosts/matmodlab/mml_userenv.py')
if not os.path.isfile(mml_userenv):
  log_warning('Kayenta user environment file not found')
else:
  setenv('MML_USERENV', mml_userenv)
